src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -382,6 +401,9 @@ public JdbcSinkConfig(Map<?, ?> props) {
     pkFields = getList(PK_FIELDS);
     dialectName = getString(DIALECT_NAME_CONFIG);
     fieldsWhitelist = new HashSet<>(getList(FIELDS_WHITELIST));
+    String dbTimeZone = getString(DB_TIMEZONE_CONFIG);
+    timeZone = dbTimeZone == null ? TimeZone.getTimeZone(ZoneOffset.UTC) :
+        TimeZone.getTimeZone(ZoneId.of(dbTimeZone));
```

Comment:
The `getString(DB_TIMEZONE_CONFIG)` method will always return a non-null string, since the default value `is `UTC`. That means that lines 405-406 can be simplified to just:

    timeZone = TimeZone.getTimeZone(ZoneId.of(dbTimeZone))

Also, what happens if an invalid time zone is specified? We can prevent that by adding a Validator to the DB_TIMEZONE_CONFIG definition that checks whether the supplied value is a valid time zone name.

Reply:
you're right for simplifying the retrieval of the timezone. thanks for the hint.

regarding an invalid timezone i'll include a validator in the configuration of the timezone and add a corresponding unit test.